tracer/test/snapshots/WcfTests.WebHttp_v1.webHttp.verified.txt

```diff
@@ -0,0 +1,141 @@
+[
+  {
+    TraceId: Id_1,
+    SpanId: Id_2,
+    Name: http.server.request,
+    Resource: ServerSyncAddJson/{n1}/n2={n2},
```

Comment:
Should we prepend a `/` to the route? I think the answer is no, but wanted to see if you had already considered this

Reply:
Hmmm, no, I didn't consider that 🤔 But that would be consistent with the aspnet integrations in which we always add them, so I think we should, thanks!

Reply:
Another point in favor is that if we _don't_ enable the feature flag, they _do_ have the leading slash, so I _definitely_ think we should add it. Thanks!